### PR TITLE
documentation for designate-manage

### PR DESCRIPTION
**Pull request by sjamgade**

### CODE REVIEW

File: designate.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
+and records over the network using pools of DNS server. From an Admin's perspective there is not
```
Comment (KeithMnemonic) [inline designate.md]: servers

File: designate.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+much to configure as the default for deployment are well in place.
```
Comment (KeithMnemonic) [inline designate.md]: I believe there are steps needed to enable the neutron integration. Is the barclamp doing this automatically?

https://docs.openstack.org/designate/latest/contributor/integrations.html
https://docs.openstack.org/ocata/networking-guide/config-dns-int.html

File: designate.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+failures.
+
+- designate-server role
+this role will install the designate server packages and configure mini-dns (mdns) service required
+by designate
+
+- designate-worker role
+this role configures a designate worker on the selected nodes. Designate uses the workers to distribute
+its workload
+
```
Comment (KeithMnemonic) [inline designate.md]: if you are not supporting Sink, (a good idea IMHO) , should we mention Sink is not included

Comment (sjamgade) [reply designate.md]: yes we should atleast mention it

File: designate.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
```
Comment (asettle) [inline designate.md]: SUSE OpenStack Cloud uses Designate (DNS as a Service) to create and propgate zones and records over the network...

File: designate.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+and records over the network using pools of DNS servers. From an Admin's perspective there is not
```
Comment (asettle) [inline designate.md]: From an administrator's perspective*

File: designate.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+much to configure as the default for deployment are well in place. Neutron needs some additional
+settings for integration with desginate[1], the barclamp currently does not configure these but if
```
Comment (asettle) [inline designate.md]: Designate*

File: designate.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+required the admin can enable these as required.
+
+This designate barclamp relies heavily on dns barclamp and expects it to be applied without any
```
Comment (asettle) [inline designate.md]: on the DNS barclamp

File: designate.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+failures.
+
+- designate-server role
+this role will install the designate server packages and configure mini-dns (mdns) service required
```
Comment (asettle) [inline designate.md]: This role installs the Designate *

File: designate.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+by designate
+
+- designate-worker role
+this role configures a designate worker on the selected nodes. Designate uses the workers to distribute
```
Comment (asettle) [inline designate.md]: This role configures a Designate worker*

File: designate.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+its workload
```
Comment (asettle) [inline designate.md]: Missing a full stop

**Comment by csymons-suse** [comment]
> You can probably ignore my comments for language changing :) we can fix that!
> Makes sense to me. @csymons-suse is there a JIRA card for this? If not, I'm not entirely too sure on our process. Do we need a JIRA card to track these changes? And then, what's our process for converting into XML?

As noted in an earlier (the first) comment...
"relates to SCRD-8739; not complete yet"
https://jira.suse.de/browse/SCRD-8739
I'm working with Sumit on this. He will assign me as worker when the content is ready. Consequently, until I hear from him, I'm assuming that it's not ready.

Until final copy hits GitHub, all activity and progress is tracked in Jira. It has an extensive communication structure that is superior to GitHub for SUSE projects.

In this case, the process for converting into XML...
- git fetch the original PR
- use pandoc to convert from Markdown to DocBook5
- do the editing thing; there is little need to do any editing
    prior to this. Doc Writer just needs the basic content
    and then can polish, request clarification & information.
- create a new PR with the DocBook file and associated PNG file...
    noting that the new PR supersedes the original PR
    request review from the original author and anyone else involved
- close the original PR with comment about superseding PR

File: designate.md (hunk 6 above)
Comment (nicolasbock) [inline designate.md]: Spelling: `s/desginate/Designate/`

File: designate.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
+Deploying Designate
+-------------------
+
+Designate provides SUSE OpenStack Cloud, DNS as a service. It allows creating and propagating zones
+and records over the network using pools of DNS servers. From an Admin's perspective there is not
+much to configure as the default for deployment are well in place. Neutron needs some additional
+settings for integration with desginate[1], the barclamp currently does not configure these but if
+required the admin can enable these as required.
+
```
Comment (nicolasbock) [inline designate.md]: Maybe go into some more details as to how one would do that. Can the user just add a drop in configuration file for Neutron? Or are there hidden settings in the Neutron barclamp that the user can use?

**Comment by jeremy-moffitt** [comment]
is this for https://jira.suse.de/browse/SCRD-8739 ? or another ticket

**Comment by csymons-suse** [comment]
> is this for [jira.suse.de/browse/SCRD-8739](https://jira.suse.de/browse/SCRD-8739) ? or another ticket

yes, SCRD-8739

**Comment by asettle** [comment]
> > You can probably ignore my comments for language changing :) we can fix that!
> > Makes sense to me. @csymons-suse is there a JIRA card for this? If not, I'm not entirely too sure on our process. Do we need a JIRA card to track these changes? And then, what's our process for converting into XML?
> 
> As noted in an earlier (the first) comment...
> "relates to SCRD-8739; not complete yet"
> https://jira.suse.de/browse/SCRD-8739
> I'm working with Sumit on this. He will assign me as worker when the content is ready. Consequently, until I hear from him, I'm assuming that it's not ready.
> 
> Until final copy hits GitHub, all activity and progress is tracked in Jira. It has an extensive communication structure that is superior to GitHub for SUSE projects.
> 
> In this case, the process for converting into XML...
> 
> * git fetch the original PR
> * use pandoc to convert from Markdown to DocBook5
> * do the editing thing; there is little need to do any editing
>   prior to this. Doc Writer just needs the basic content
>   and then can polish, request clarification & information.
> * create a new PR with the DocBook file and associated PNG file...
>   noting that the new PR supersedes the original PR
>   request review from the original author and anyone else involved
> * close the original PR with comment about superseding PR

THANK YOU CARL! :D

**Comment by csymons-suse** [comment]
when complete will be superseded by 
https://github.com/SUSE-Cloud/doc-cloud/pull/931
please advise when content for this PR is complete

**Comment by csymons-suse** [comment]
superseded by https://github.com/SUSE-Cloud/doc-cloud/pull/931

**Pull request closed without merging**